.eslintrc.js

```diff
@@ -42,12 +42,13 @@ module.exports = {
       },
     },
     {
-      files: ["packages/app/test/**/*.{js,ts}"],
+      files: ["packages/app/test/**/*.{js,ts}", "*.docs.tsx"],
       rules: {
         // we don't want to install cypress
         // in production builds
         "import/no-extraneous-dependencies": "off",
         "global-require": "off",
+        "no-console": "off",
```

Comment:
Hope this one's temporary for debug : D

Reply:
I decided that logging stuff to console docs is okay, and in end-to-end tests, I just don't care that much. Splitting this group into two might make sense, though.